R/get_paths.R

```diff
@@ -316,7 +316,7 @@ get_fromids <- function(index_ids, return_list = FALSE) {
 #' @noRd
 #' @param x data.frame with an identifier and to identifier in the
 #' first and second columns.
-get_sorted <- function(x) {
+get_sorted <- function(x, split = FALSE, return_list = FALSE) {
```

Comment:
add params to documentation for split and return_list?

Reply:
Will do -- and export.